blueprints/ember-cli-eslint/files/.eslintrc.js

```diff
@@ -1,3 +1,7 @@
+var path = require('path');
+
 module.exports = {
-  extends: './node_modules/ember-cli-eslint/coding-standard/ember-application.js'
+  extends: [
+    path.join(__dirname, 'node_modules/ember-cli-eslint/coding-standard/ember-application.js')
```

Comment:
@rwjblue i think resolve is relative `cwd` whereas path.join(__dirname is relative to the current files dirname.